src/pages/patientView/structuralVairnt/PatientViewStructuralVariantTable.tsx

```diff
@@ -0,0 +1,403 @@
+        });
+
+        columns.push({
+            name: 'Site1 Position',
+            render: (d: StructuralVariant[]) => (
+                <span>{d[0].site1Position}</span>
+            ),
+            download: (d: StructuralVariant[]) => `${d[0].site1Position}`,
+            sortBy: (d: StructuralVariant[]) => `${d[0].site1Position}`,
+            visible: false,
+            order: 55,
+        });
+
+        columns.push({
+            name: 'Site2 Position',
+            render: (d: StructuralVariant[]) => (
+                <span>{d[0].site2Position}</span>
+            ),
+            download: (d: StructuralVariant[]) => `${d[0].site2Position}`,
+            sortBy: (d: StructuralVariant[]) => `${d[0].site2Position}`,
+            visible: false,
+            order: 65,
+        });
+
+        columns.push({
+            name: 'Event Info',
+            render: (d: StructuralVariant[]) => <span>{d[0].eventInfo}</span>,
+            download: (d: StructuralVariant[]) => d[0].eventInfo,
+            sortBy: (d: StructuralVariant[]) => d[0].eventInfo,
+            visible: false,
+            order: 66,
+        });
+
+        columns.push({
+            name: 'Connection Type',
+            render: (d: StructuralVariant[]) => (
+                <span>{d[0].connectionType}</span>
+            ),
+            download: (d: StructuralVariant[]) => d[0].connectionType,
+            sortBy: (d: StructuralVariant[]) => d[0].connectionType,
+            visible: true,
+            order: 70,
+        });
+
+        columns.push({
+            name: 'Breakpoint Type',
+            render: (d: StructuralVariant[]) => (
+                <span>{d[0].breakpointType}</span>
+            ),
+            download: (d: StructuralVariant[]) => d[0].breakpointType,
+            sortBy: (d: StructuralVariant[]) => d[0].breakpointType,
+            visible: false,
+            order: 75,
+        });
+
+        columns.push({
+            name: 'Additional Annotation',
+            render: (d: StructuralVariant[]) => <span>{d[0].annotation}</span>,
+            download: (d: StructuralVariant[]) => d[0].annotation,
+            sortBy: (d: StructuralVariant[]) => d[0].annotation,
+            visible: false,
+            order: 80,
+        });
+
+        return _.sortBy(columns, (c: CNATableColumn) => c.order);
+    }
+
+    readonly tableUI = MakeMobxView({
+        await: () => [
+            this.props.store.groupedStructuralVariantData,
+            this.props.store.oncoKbAnnotatedGenes,
+            this.props.store.studyIdToStudy,
+            this.props.store.sampleManager,
+            this.props.store.structuralVariantTableShowGeneFilterMenu,
+            this.props.store.sampleToStructuralVariantGenePanelId,
+            this.props.store.genePanelIdToEntrezGeneIds,
+        ],
+        render: () => {
+            if (
+                this.props.store.structuralVariantProfileId.result === undefined
+            ) {
+                return (
```

Comment:
can we be sure this message won't show in pending state?  i guess if mobx promise has default.  but would be better to check for both isComplete and result undefined (if indeed the result can be undefined when call has completed

Reply:
I agree ^^

Reply:
`groupedStructuralVariantData` is in await, which already awaits for `structuralVariantProfileId`. Wouldn't it be redundant to add again?